src/generators/elixir.ts

```
@@ -27,27 +26,21 @@ const supportedArgs = new Set([
   "user",
 ]);
 
-// TODO: I bet elixir's array syntax is different and if the query string
-// values are arrays that actually generates broken code.
 function repr(value: string | null | (string | null)[]): string {
   // In context of url parameters, don't accept nulls and such.
-  if (!value) {
-    return '""';
-  } else {
-    return `~s|${jsesc(value, { quotes: "backtick" })}|`;
-  }
+  return '"' + jsesc(value, { quotes: "double", minimal: true }) + '"';
 }
 
 function getCookies(request: Request): string {
-  if (!request.cookies) {
+  if (!request.cookies || !request.cookies.length) {
     return "";
   }
 
   const cookies = [];
   for (const [cookieName, cookieValue] of request.cookies) {
     cookies.push(`${cookieName}=${cookieValue}`);
   }
-  return `cookies: [~s|${cookies.join("; ")}|]`;
+  return `cookie: [${repr(cookies.join("; "))}]`;
```

This one is kind of funny, `cookies:` is silently ignored